backend/src/db-utils.js

```diff
@@ -529,6 +529,38 @@ const queryPartnerUniqueUserAmount = async () => {
   );
 };
 
+const getLockupAccountIds = async (blockHeight) => {
+  return await queryRows(
+    [
+      `SELECT accounts.account_id
+       FROM accounts
+                LEFT JOIN receipts AS receipts_start ON accounts.created_by_receipt_id = receipts_start.receipt_id
+                LEFT JOIN blocks AS blocks_start ON receipts_start.included_in_block_hash = blocks_start.block_hash
+                LEFT JOIN receipts AS receipts_end ON accounts.deleted_by_receipt_id = receipts_end.receipt_id
+                LEFT JOIN blocks AS blocks_end ON receipts_end.included_in_block_hash = blocks_end.block_hash
+       WHERE accounts.account_id like '%.lockup.near'
+         AND (blocks_start.block_height IS NULL OR blocks_start.block_height <= :blockHeight)
+         AND (blocks_end.block_height IS NULL OR blocks_end.block_height >= :blockHeight);`,
+      { blockHeight: blockHeight },
+    ],
+    { dataSource: DS_INDEXER_BACKEND }
+  );
+};
+
+const getLastYesterdayBlock = async (timestamp) => {
+  return await querySingleRow(
+    [
+      // we can't cast timestamp to number (it's too big), BN is not OK here, that's why I concatenate strings
+      `SELECT block_height
+       FROM blocks
+       WHERE block_timestamp < ` +
+        timestamp.toString() +
+        `\nORDER BY block_timestamp DESC LIMIT 1;`,
```

Comment:
This should work, I believe:

```suggestion
       WHERE block_timestamp < :blockTimestamp
       ORDER BY block_timestamp DESC LIMIT 1;`,
       { blockTimestamp: timestamp.toString() },
```

Reply:
Nope, there are quotes 😞